include/z64scene.h

```diff
@@ -74,87 +74,91 @@ typedef struct {
 typedef struct {
     /* 0x00 */ Gfx* opa;
     /* 0x04 */ Gfx* xlu;
-} MeshHeader01Entry; // size = 0x08
+} MeshHeaderDListsEntry; // size = 0x08
 
 typedef struct {
     /* 0x00 */ MeshHeaderBase base;
     /* 0x01 */ u8 numEntries;
-    /* 0x04 */ MeshHeader01Entry* entries;
-    /* 0x08 */ MeshHeader01Entry* entriesEnd;
-} MeshHeader0; // size = 0x0C
+    /* 0x04 */ MeshHeaderDListsEntry* entries;
+    /* 0x08 */ MeshHeaderDListsEntry* entriesEnd;
+} MeshHeaderAll; // size = 0x0C
```

Comment:
I think `All` is an unfortunate adjective to use since it could be interpreted as encompassing the others, rather than being a distinct type that draws the whole lot. I don't have a better suggestion yet, will think about it more.

Reply:
thought this as well when skimming the pr for the first time, fwiw

Reply:
I admit I didn't make a big effort to name this one, but the only other idea I had was "Simple"/"Normal"/"Basic" :sweat: 
"Everything" seemed worse too
If anyone can come up with a suggestion it's taken

Reply:
all supposed to convey that there is no conveying and all of them are drawn at all times right?
if my understanding is correct then Normal is probably best for me, but also open to other suggestions

Reply:
Changed to "Normal" but I'm not a fan :shrug: 
wasn't a fan of "All" either anyway

If anyone has a suggestion eventually, feel free